sljit_src/sljitNativeMIPS_common.c

```diff
@@ -324,6 +323,8 @@ static const sljit_u8 freg_map[SLJIT_NUMBER_OF_FLOAT_REGISTERS + 4] = {
 #define LOAD_W		LD
 #endif
 
+#define MOV_fmt(f)	(HI(17) | f | LO(6))
```

Comment:
All other float instructions use the `_S` postfix (because fmt s is zero). Do you plan to change all of them?

Reply:
`FMT_S` is actually 16 and all had been changed already as part of the "refactoring" done here

Reply:
I am talking about the other float instructions such as add.fmt, sub.fmt, etc.